packages/charts-react-line/src/LineChart.js

```diff
@@ -72,13 +86,21 @@ export default class LineChart extends React.PureComponent {
 
   render() {
     const { line, props } = this;
-    const { data, margin, width, height } = props;
+    const { data, margin, strokes, width, height } = props;
+    const lines = this.props.lines.map((data, index) =>
+      <Line
+        line = {line}
+        data = {data}
+        key={index}
+        stroke={strokes[index % strokes.length]}
```

Comment:
Is there a reason why we need to cycle through the stroke colors for consumers? Could they decide the color, and we just provide the color presets that they could use?